### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by RussellAnderson**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/RussellAnderson/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome
```
Comment (RussellAnderson) [inline ReadMe.md:1]: commenting on the line !fix

Comment (RussellAnderson) [inline ReadMe.md:1]: !fixed

**Comment by RussellAnderson** [comment]
interesting

File: ReadMe.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+
+**Thanks for letting us show you around!**
```
Comment (RussellAnderson) [inline ReadMe.md:3]: looks good

**Pull request merged**